src/ttsim/interface_dag_elements/policy_environment.py

```diff
@@ -162,33 +161,35 @@ def _get_one_param(  # noqa: PLR0911
         )
         return PiecewisePolynomialParam(**cleaned_spec)
     if spec["type"] == "consecutive_int_1d_lookup_table":
```

Comment:
We should get rid of the 1d / 2d here, too. In particular, we should be able to infer the depth from the nesting level of the dictionary in `cleaned_spec["value"]`